db/migrate/20170817064448_create_tenant_api_requests.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTenantApiRequests < ActiveRecord::Migration
+  def change
+    create_table :tenant_api_requests do |t|
```

Comment:
The only issue i faced with the previous solution is to track the requests count of 100 every day which was not possible with it, So i choose to save in different table with track date as separate column and has_many association with tenant, so in that way i can track 100 requests every day, and apply throttle after that for 1 request per 10 seconds